Viper/Viper/HotelsModule/View/HotelsViewController.swift

```diff
@@ -0,0 +1,72 @@
+// HotelsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран отелей
+final class HotelsViewController: UIViewController, HotelsViewProtocol {
+    // MARK: - Private Visual Components
+
+    private var tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.separatorStyle = .none
+        tableView.register(HotelTableViewCell.self, forCellReuseIdentifier: S.tableCellIdentifier)
+        return tableView
+    }()
+
+    // MARK: - Public property
+
+    var presenter: HotelsPresenterProtocol?
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private methods
+
+    private func setupUI() {
+        view.addSubview(tableView)
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.backgroundColor = .white
+        if let country = presenter?.country {
+            navigationItem.title = "Отели в \(country)"
+        }
```

Comment:
```suggestion
        view.addSubview(tableView)
        tableView.dataSource = self
        tableView.delegate = self
        view.backgroundColor = .white
        if let country = presenter?.country {
            navigationItem.title = "Отели в \(country)"
        }
```
тут тоже в куче можно разнести по методам, хотя бы свойства таблицы